server/plugin/api.go

```diff
@@ -413,6 +413,21 @@ func (p *Plugin) handleGetSubscriptions(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	projectList, err := p.Store.GetAllProjects(mattermostUserID)
+	if err != nil {
+		p.API.LogError(constants.ErrorFetchProjectList, "Error", err.Error())
+		p.handleError(w, r, &serializers.Error{Code: http.StatusInternalServerError, Message: err.Error()})
+		return
+	}
+
+	organization := pathParams[constants.PathParamOrganization]
+	project := pathParams[constants.PathParamProject]
```

Comment:
We could "return early" if `project` is blank right? Before we interact with the store

Reply:
Yes, we have made `organization` and `project` path params so, if they are empty this API will not be called as they will not reach here and get `404`